### ISSUE TITLE
Possible selectedColor attributes for Label 

### ISSUE THREAD

**Issue by atakanokan** [heartexlabs/label-studio#625]
The documentation doesn't state what are the acceptable arguments as strings to the `selectedColor` attribute for `Label` in `config.xml`. 

https://github.com/heartexlabs/label-studio/blob/e34e4cd3a9f4361b5f3b292eafecff14f305fd36/docs/source/tags/label.md

![image](https://user-images.githubusercontent.com/20222221/109409912-64779a00-794b-11eb-9be0-10dcaca437de.png)

**Comment by smoreface** [comment]
@hlomzik can you confirm that the colors in https://github.com/heartexlabs/label-studio-frontend/blob/master/src/utils/colors.js are the available options for this field?

**Comment by atakanokan** [comment]
I just tried re-starting a project with multiple `Label`s such as  `<Label value="X" selectedColor="green"/>`  and the color changes seem to not take any effect. (Any other change such as adding a new `Label` would be in effect with a new start of the project, not init.)

**Comment by atakanokan** [comment]
@smoreface Can we add info about the `background` attribute too?

**Comment by smoreface** [comment]
@atakanokan yes, I can do that. Glad to hear the color name works with the background parameter!
